qemu/tests/cfg/blockdev_mirror_cancel_ready_job_with_io.cfg

```diff
@@ -30,7 +30,8 @@
     image_name_mirror1 = mirror1
 
     nbd:
-        nbd_port_data1 = 10822
```

Comment:
10822 is used for a 2g qcow2 nbd image, so here do we want to add that? e.g.
qcow2:
    image_format_data1 = qcow2
    nbd_port_data1 = 10822
raw:
   nbd_port_data1 = 10831
   image_format_data1 = raw

Reply:
I noticed that we have several same changes(10822->10831) for the following cfg files, please also consider those changes if you agree to add qcow2 support, I will not leave comments for those changes, thanks.